MANIFEST.in

```diff
@@ -3,4 +3,5 @@ include CHANGELOG.rst
 include CONTRIBUTING.rst
 include LICENSE.txt
 include README.rst
+include requirements/base.in
```

Comment:
What is the role of adding / including the requirements file here?

Reply:
By including this file in the Manifest.in, the file will become available in the distribution package. It was needed as without it, the `setup.py` was not able to locate the requirements/base.in when defining install_requires through function.